build/Targets/VSL.Versions.targets

```diff
@@ -73,8 +73,8 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <CommonNuGetMoniker>beta2</CommonNuGetMoniker>
-    <VSSemanticVersion>15.3</VSSemanticVersion>    
+    <CommonNuGetMoniker>beta3</CommonNuGetMoniker>
```

Comment:
Maybe? We only have instructions for how to create branches in the roslyn and roslyn-internal repos. It isn't always obvious how to translate those to other repos.